api/app/hfi/prep_pdf_gen.py

```diff
@@ -0,0 +1,26 @@
+"""Generate a prep cycle PDF"""
+import pdfkit
+from jinja2 import Environment, FunctionLoader
+from app.hfi.prep_template import str_prep_template
+
+
+jinja_env = Environment(loader=FunctionLoader(str_prep_template), autoescape=True)
```

Comment:
This python module can be merged with the daily one so that we have one PDF generated. If you want to address this in a follow up PR that's cool with me.

Reply:
Yeah that was my idea

Reply:
Can we throw a `#TODO` to that affect in here please?